Overlay: add evox-specific and sfps props

weaponmasterjax opened the pull request.

mmtrt commented:
can you use only rro_overlays for evox specific overlay changes, use separate rro_overlays with all previous non evox overlay changes with added evox changes and cutoff non evox overlay of these with check.

```
FrameworksResEvoxOverlayEverpal
SettingsEvoxOverlayEverpal
```

weaponmasterjax commented:
done.

mmtrt commented on device.mk:
```diff
@@ -431,13 +431,23 @@ PRODUCT_PACKAGES += \
+    PRODUCT_PACKAGES += \
+        EvolutionSettingsResEverpal \
+        EvolutionFrameworksResEverpal \
+        FrameworksResOverlayEverpal \
```
why add these here I said use them as base overlay for evox containing all changes including evox one

weaponmasterjax committed:
[squash]Evox: add evox specific overlay


Add maintainer info


Overlay: add evox-specific and sfps props


nuck udfps


Move Evox specific overlay to rro_overlay


Move non-Evox RROs


Make evox RROs standalone

weaponmasterjax committed:
revert non evox changes
